extensions/lifecycle/tasks/LifecycleTaskV2.js

```diff
@@ -255,7 +257,9 @@ class LifecycleTaskV2 extends LifecycleTask {
                     return resolve();
                 })));
 
-            return Promise.allSettled(promises).then(() => done(), done);
+            return Promise.allSettled(promises).then(results => done(
+                results.findLast(r => r.status === 'rejected')[0]?.reason
```

Comment:
Shouldn't it be

```suggestion
                results.findLast(r => r.status === 'rejected')?.reason
```

as it an array of dimension 1, and findLast only returns the first matching element?

Reply:
yes, switch from filter to findLast on the last minute...